lua/nvim-treesitter/parsers.lua

```diff
@@ -557,6 +612,10 @@ function M.ft_to_lang(ft)
   return ft_to_parsername[ft] or ft
 end
 
+function M.alias_to_lang(alias)
+  return injections_aliases[alias]
+end
+
```

Comment:
Might be a good idea to add a function to allow user's to add custom alias' too.

```suggestion

function M.add_injection_alias(alias, lang)
  injection_aliases[alias] = lang
end
```